packages/asc-ui/src/components/SocialBar/SocialBar.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import { AscCore } from '@datapunt/asc-core'
+import { ReactComponent as Facebook } from '@datapunt/asc-assets/lib/Icons/Facebook.svg'
+import { ReactComponent as Twitter } from '@datapunt/asc-assets/lib/Icons/Twitter.svg'
+import { ReactComponent as Linkedin } from '@datapunt/asc-assets/lib/Icons/Linkedin.svg'
+import { ReactComponent as Email } from '@datapunt/asc-assets/lib/Icons/Email.svg'
+import { ReactComponent as Print } from '@datapunt/asc-assets/lib/Icons/Print.svg'
+import SocialButton from '../SocialButton'
+
+const SocialBar: React.FC<{}> = ({ children }) => (
+  <AscCore.SocialBar>
```

Comment:
I don't think we should pre-define all the socialbuttons in this component, this is what the user should do

Reply:
Well, it gives developers an idea of what they can use.

Reply:
Then I think it should be moved to a story / documentation example rather than a standalone component

Reply:
Moved implementation to a story example